cmd/kubeadm/app/util/system/types_unix.go

```diff
@@ -30,7 +30,7 @@ const dockerEndpoint = "unix:///var/run/docker.sock"
 var DefaultSysSpec = SysSpec{
 	OS: "Linux",
 	KernelSpec: KernelSpec{
-		Versions: []string{`3\.[1-9][0-9].*`, `4\..*`, `5\..*`}, // Requires 3.10+, 4+ or 5+
+		Versions: []string{`3\.[1-9][0-9].*`, `[4-99]\..*`}, // Requires 3.10+, or newer
```

Comment:
should this and the other regex just be defined in one place instead of both meaning we have to copy and paste it wherever we want it? can we make it a package const or something?

Reply:
can be done, but while this regex should hold the actual values, the other one can have any values, as it's just a test.

if we have more votes for this i can update it.

Reply:
@neolit123 - if we need always to change both at the same, it seems reasonable to have const for this package

Reply:
If versions between 4.x and 99.x is what you desire, then `[4-99]\..*` is pretty wrong. This one should do the trick in that case:
`[4-9]\..*|[1-9][0-9]\..*`

And let's hope we don't get Linux 99.x soon.

Reply:
> Any version >= 4.x is going to be [4-9]\..*|[1-9][0-9]*\..*, then Linux 999.x would be OK too.

ok, i will give this a test.

> And let's hope we don't get Linux 99.x soon.

i think i calculated roughly 25 more years of Linux kernels with the current cadence until we reach 99.x

Reply:
+1 

`[4-99]` would match the  4-9 range and 9

Reply:
updated with the correct regexp, also there was a bug in the old regexp for 3* versions :)

i kept the regexp between _unix and the test file duplicated.
the correct thing to do is to introduce separate _windows and _unix unit tests for kernel versions, but i don't wish to add this as part of this PR.

Reply:
tests https://play.golang.org/p/QaO4uCZBE2v